charts/akeyless-api-gateway/Chart.yaml

```diff
@@ -19,4 +19,4 @@ version: 1.13.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 3.12.0
+appVersion: 3.13.0
```

Comment:
you need to bump the chart version as well

Reply:
also bump minor version

Reply:
update chart version to 1.14.0